2014/07/24 09:43:38 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:43:50 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:43:54 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:43:54 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:43:59 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:44:11 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:44:15 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:44:15 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:44:19 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:44:31 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:44:35 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:44:35 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:44:39 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:44:51 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:44:55 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:44:55 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:44:59 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:45:11 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:45:15 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:45:15 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:45:19 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:45:31 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:45:35 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:45:35 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:45:39 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:45:51 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:45:55 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:45:55 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:45:59 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:46:11 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:46:15 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:46:15 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:46:19 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:46:31 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:46:35 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:46:35 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:46:39 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:46:51 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:46:55 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:46:55 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:46:59 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:47:11 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:47:15 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:47:15 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:47:19 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:47:31 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:47:35 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:47:35 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:47:40 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:47:52 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:47:56 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:47:56 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:48:00 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:48:12 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:48:16 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:48:16 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:48:20 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:48:32 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:48:36 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:48:36 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:48:40 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:48:52 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:48:56 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:48:56 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:49:00 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:49:12 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:49:16 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:49:16 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:49:20 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:49:32 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:49:36 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:49:36 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:49:40 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:49:52 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:49:56 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:49:56 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:50:00 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:50:12 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:50:16 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:50:16 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:50:20 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:50:32 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:50:36 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:50:36 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:50:40 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:50:52 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:50:56 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:50:56 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:51:00 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:51:12 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:51:16 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:51:16 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:51:20 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:51:32 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:51:36 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:51:37 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:51:41 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:51:53 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:51:57 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:51:57 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:52:01 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:52:13 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:52:17 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:52:17 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:52:21 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:52:33 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:52:37 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:52:37 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:52:41 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:52:53 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:52:57 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:52:57 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:53:01 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:53:13 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:53:17 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:53:17 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:53:21 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:53:33 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:53:37 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:53:37 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:53:41 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:53:55 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:54:00 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:54:00 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:54:05 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:54:20 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:54:25 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:54:25 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:54:30 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:54:45 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:54:52 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405140023.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405140023.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 09:54:52 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_47_1405138619.jpg".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_47_1405138619.jpg
HTTP_REFERER=http://localhost/mypropertylk/list/detail/pid/47
---
2014/07/24 10:00:17 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[HY000]: General error: 1452 Cannot add or update a child row: a foreign key constraint fails (`mypropertylk`.`property`, CONSTRAINT `property_ibfk_4` FOREIGN KEY (`district`) REFERENCES `district` (`id`) ON DELETE CASCADE). The SQL statement executed was: INSERT INTO `property` (`type`, `sendemail`, `hidestreetaddress`, `dispalyprice`, `propcondition`, `pricetype`, `entrydate`, `agent`, `otheragent`, `price`, `vendorname`, `vendoremail`, `vendorphone`, `unitnum`, `number`, `streetaddress`, `areaname`, `townname`, `district`, `province`, `owner`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6, :yp7, :yp8, :yp9, :yp10, :yp11, :yp12, :yp13, :yp14, :yp15, :yp16, :yp17, :yp18, :yp19, :yp20).
in C:\xampplite\htdocs\mypropertylk\protected\controllers\Property\AddPropertyAction.php (69)
in C:\xampplite\htdocs\mypropertylk\protected\controllers\PropertyController.php (74)
in C:\xampplite\htdocs\mypropertylk\index.php (15)
2014/07/24 10:00:17 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[HY000]: General error: 1452 Cannot add or update a child row: a foreign key constraint fails (`mypropertylk`.`property`, CONSTRAINT `property_ibfk_4` FOREIGN KEY (`district`) REFERENCES `district` (`id`) ON DELETE CASCADE). The SQL statement executed was: INSERT INTO `property` (`type`, `sendemail`, `hidestreetaddress`, `dispalyprice`, `propcondition`, `pricetype`, `entrydate`, `agent`, `otheragent`, `price`, `vendorname`, `vendoremail`, `vendorphone`, `unitnum`, `number`, `streetaddress`, `areaname`, `townname`, `district`, `province`, `owner`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6, :yp7, :yp8, :yp9, :yp10, :yp11, :yp12, :yp13, :yp14, :yp15, :yp16, :yp17, :yp18, :yp19, :yp20)' in C:\xampplite\htdocs\mypropertylk\protected\framework\db\CDbCommand.php:354
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\db\ar\CActiveRecord.php(1014): CDbCommand->execute()
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\db\ar\CActiveRecord.php(787): CActiveRecord->insert(NULL)
#2 C:\xampplite\htdocs\mypropertylk\protected\controllers\Property\AddPropertyAction.php(69): CActiveRecord->save(false)
#3 C:\xampplite\htdocs\mypropertylk\protected\framework\web\actions\CAction.php(75): AddPropertyAction->run()
#4 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CController.php(309): CAction->runWithParams(Array)
#5 C:\xampplite\htdocs\mypropertylk\protected\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(AddPropertyAction))
#6 C:\xampplite\htdocs\mypropertylk\protected\controllers\PropertyController.php(74): CFilterChain->run()
#7 C:\xampplite\htdocs\mypropertylk\protected\framework\web\filters\CInlineFilter.php(59): PropertyController->filterAccessControl(Object(CFilterChain))
#8 C:\xampplite\htdocs\mypropertylk\protected\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#9 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CController.php(292): CFilterChain->run()
#10 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CController.php(266): CController->runActionWithFilters(Object(AddPropertyAction), Array)
#11 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(276): CController->run('addproperty')
#12 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('property/addpro...')
#13 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#14 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#15 {main}
REQUEST_URI=/mypropertylk/property/addproperty?type=1
HTTP_REFERER=http://localhost/mypropertylk/property/addproperty?type=1
---
2014/07/24 10:01:48 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[HY000]: General error: 1452 Cannot add or update a child row: a foreign key constraint fails (`mypropertylk`.`property`, CONSTRAINT `property_ibfk_4` FOREIGN KEY (`district`) REFERENCES `district` (`id`) ON DELETE CASCADE). The SQL statement executed was: INSERT INTO `property` (`type`, `sendemail`, `hidestreetaddress`, `dispalyprice`, `propcondition`, `pricetype`, `entrydate`, `agent`, `otheragent`, `price`, `vendorname`, `vendoremail`, `vendorphone`, `unitnum`, `number`, `streetaddress`, `areaname`, `townname`, `district`, `province`, `owner`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6, :yp7, :yp8, :yp9, :yp10, :yp11, :yp12, :yp13, :yp14, :yp15, :yp16, :yp17, :yp18, :yp19, :yp20).
in C:\xampplite\htdocs\mypropertylk\protected\controllers\Property\AddPropertyAction.php (69)
in C:\xampplite\htdocs\mypropertylk\protected\controllers\PropertyController.php (74)
in C:\xampplite\htdocs\mypropertylk\index.php (15)
2014/07/24 10:01:48 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[HY000]: General error: 1452 Cannot add or update a child row: a foreign key constraint fails (`mypropertylk`.`property`, CONSTRAINT `property_ibfk_4` FOREIGN KEY (`district`) REFERENCES `district` (`id`) ON DELETE CASCADE). The SQL statement executed was: INSERT INTO `property` (`type`, `sendemail`, `hidestreetaddress`, `dispalyprice`, `propcondition`, `pricetype`, `entrydate`, `agent`, `otheragent`, `price`, `vendorname`, `vendoremail`, `vendorphone`, `unitnum`, `number`, `streetaddress`, `areaname`, `townname`, `district`, `province`, `owner`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6, :yp7, :yp8, :yp9, :yp10, :yp11, :yp12, :yp13, :yp14, :yp15, :yp16, :yp17, :yp18, :yp19, :yp20)' in C:\xampplite\htdocs\mypropertylk\protected\framework\db\CDbCommand.php:354
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\db\ar\CActiveRecord.php(1014): CDbCommand->execute()
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\db\ar\CActiveRecord.php(787): CActiveRecord->insert(NULL)
#2 C:\xampplite\htdocs\mypropertylk\protected\controllers\Property\AddPropertyAction.php(69): CActiveRecord->save(false)
#3 C:\xampplite\htdocs\mypropertylk\protected\framework\web\actions\CAction.php(75): AddPropertyAction->run()
#4 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CController.php(309): CAction->runWithParams(Array)
#5 C:\xampplite\htdocs\mypropertylk\protected\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(AddPropertyAction))
#6 C:\xampplite\htdocs\mypropertylk\protected\controllers\PropertyController.php(74): CFilterChain->run()
#7 C:\xampplite\htdocs\mypropertylk\protected\framework\web\filters\CInlineFilter.php(59): PropertyController->filterAccessControl(Object(CFilterChain))
#8 C:\xampplite\htdocs\mypropertylk\protected\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#9 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CController.php(292): CFilterChain->run()
#10 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CController.php(266): CController->runActionWithFilters(Object(AddPropertyAction), Array)
#11 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(276): CController->run('addproperty')
#12 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('property/addpro...')
#13 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#14 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#15 {main}
REQUEST_URI=/mypropertylk/property/addproperty?type=1
HTTP_REFERER=http://localhost/mypropertylk/property/addproperty?type=1
---
2014/07/24 10:27:09 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[HY000]: General error: 1452 Cannot add or update a child row: a foreign key constraint fails (`mypropertylk`.`property`, CONSTRAINT `property_ibfk_3` FOREIGN KEY (`province`) REFERENCES `province` (`id`) ON DELETE CASCADE). The SQL statement executed was: UPDATE `property` SET `pid`=:yp0, `propcondition`=:yp1, `owner`=:yp2, `agent`=:yp3, `otheragent`=:yp4, `weeklyrent`=:yp5, `monthlyrent`=:yp6, `securebond`=:yp7, `price`=:yp8, `dispalyprice`=:yp9, `availabledate`=:yp10, `vendorname`=:yp11, `vendoremail`=:yp12, `vendorphone`=:yp13, `sendemail`=:yp14, `unitnum`=:yp15, `lotnum`=:yp16, `number`=:yp17, `streetaddress`=:yp18, `areaname`=:yp19, `townname`=:yp20, `hidestreetaddress`=:yp21, `district`=:yp22, `province`=:yp23, `bedrooms`=:yp24, `bathrooms`=:yp25, `ensuites`=:yp26, `toilets`=:yp27, `parkgaragespaces`=:yp28, `parkcarpetspaces`=:yp29, `parkopenspaces`=:yp30, `livingarea`=:yp31, `housesize`=:yp32, `housesquares`=:yp33, `landsize`=:yp34, `landsquares`=:yp35, `floorarea`=:yp36, `floorsquares`=:yp37, `tenuretype`=:yp38, `building`=:yp39, `parkingspaces`=:yp40, `parkcomment`=:yp41, `zoning`=:yp42, `outgoings`=:yp43, `eer`=:yp44, `balcony`=:yp45, `deck`=:yp46, `oea`=:yp47, `shed`=:yp48, `remotegarage`=:yp49, `swimpool`=:yp50, `courtyard`=:yp51, `fullyfenced`=:yp52, `outsidespa`=:yp53, `securepark`=:yp54, `tenniscourt`=:yp55, `spabovroundeg`=:yp56, `alarmsys`=:yp57, `biltinwardrobes`=:yp58, `dvs`=:yp59, `gym`=:yp60, `intercom`=:yp61, `rumpusroom`=:yp62, `workshop`=:yp63, `broadbandinternet`=:yp64, `dishwasher`=:yp65, `floorboards`=:yp66, `insidespa`=:yp67, `paytv`=:yp68, `study`=:yp69, `ac`=:yp70, `heating`=:yp71, `cooling`=:yp72, `solarpower`=:yp73, `solarhotwater`=:yp74, `watertank`=:yp75, `otherfeatures`=:yp76, `headline`=:yp77, `desc`=:yp78, `vediourl`=:yp79, `onlinetour1`=:yp80, `onlinetour2`=:yp81, `entrydate`=:yp82, `status`=:yp83, `type`=:yp84, `pricetype`=:yp85 WHERE `property`.`pid`=47.
in C:\xampplite\htdocs\mypropertylk\protected\controllers\Property\EditPropertyAction.php (64)
in C:\xampplite\htdocs\mypropertylk\protected\controllers\PropertyController.php (74)
in C:\xampplite\htdocs\mypropertylk\index.php (15)
2014/07/24 10:27:09 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[HY000]: General error: 1452 Cannot add or update a child row: a foreign key constraint fails (`mypropertylk`.`property`, CONSTRAINT `property_ibfk_3` FOREIGN KEY (`province`) REFERENCES `province` (`id`) ON DELETE CASCADE). The SQL statement executed was: UPDATE `property` SET `pid`=:yp0, `propcondition`=:yp1, `owner`=:yp2, `agent`=:yp3, `otheragent`=:yp4, `weeklyrent`=:yp5, `monthlyrent`=:yp6, `securebond`=:yp7, `price`=:yp8, `dispalyprice`=:yp9, `availabledate`=:yp10, `vendorname`=:yp11, `vendoremail`=:yp12, `vendorphone`=:yp13, `sendemail`=:yp14, `unitnum`=:yp15, `lotnum`=:yp16, `number`=:yp17, `streetaddress`=:yp18, `areaname`=:yp19, `townname`=:yp20, `hidestreetaddress`=:yp21, `district`=:yp22, `province`=:yp23, `bedrooms`=:yp24, `bathrooms`=:yp25, `ensuites`=:yp26, `toilets`=:yp27, `parkgaragespaces`=:yp28, `parkcarpetspaces`=:yp29, `parkopenspaces`=:yp30, `livingarea`=:yp31, `housesize`=:yp32, `housesquares`=:yp33, `landsize`=:yp34, `landsquares`=:yp35, `floorarea`=:yp36, `floorsquares`=:yp37, `tenuretype`=:yp38, `building`=:yp39, `parkingspaces`=:yp40, `parkcomment`=:yp41, `zoning`=:yp42, `outgoings`=:yp43, `eer`=:yp44, `balcony`=:yp45, `deck`=:yp46, `oea`=:yp47, `shed`=:yp48, `remotegarage`=:yp49, `swimpool`=:yp50, `courtyard`=:yp51, `fullyfenced`=:yp52, `outsidespa`=:yp53, `securepark`=:yp54, `tenniscourt`=:yp55, `spabovroundeg`=:yp56, `alarmsys`=:yp57, `biltinwardrobes`=:yp58, `dvs`=:yp59, `gym`=:yp60, `intercom`=:yp61, `rumpusroom`=:yp62, `workshop`=:yp63, `broadbandinternet`=:yp64, `dishwasher`=:yp65, `floorboards`=:yp66, `insidespa`=:yp67, `paytv`=:yp68, `study`=:yp69, `ac`=:yp70, `heating`=:yp71, `cooling`=:yp72, `solarpower`=:yp73, `solarhotwater`=:yp74, `watertank`=:yp75, `otherfeatures`=:yp76, `headline`=:yp77, `desc`=:yp78, `vediourl`=:yp79, `onlinetour1`=:yp80, `onlinetour2`=:yp81, `entrydate`=:yp82, `status`=:yp83, `type`=:yp84, `pricetype`=:yp85 WHERE `property`.`pid`=47' in C:\xampplite\htdocs\mypropertylk\protected\framework\db\CDbCommand.php:354
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\db\ar\CActiveRecord.php(1681): CDbCommand->execute()
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\db\ar\CActiveRecord.php(1061): CActiveRecord->updateByPk('47', Array)
#2 C:\xampplite\htdocs\mypropertylk\protected\framework\db\ar\CActiveRecord.php(787): CActiveRecord->update(NULL)
#3 C:\xampplite\htdocs\mypropertylk\protected\controllers\Property\EditPropertyAction.php(64): CActiveRecord->save(false)
#4 C:\xampplite\htdocs\mypropertylk\protected\framework\web\actions\CAction.php(75): EditPropertyAction->run()
#5 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CController.php(309): CAction->runWithParams(Array)
#6 C:\xampplite\htdocs\mypropertylk\protected\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(EditPropertyAction))
#7 C:\xampplite\htdocs\mypropertylk\protected\controllers\PropertyController.php(74): CFilterChain->run()
#8 C:\xampplite\htdocs\mypropertylk\protected\framework\web\filters\CInlineFilter.php(59): PropertyController->filterAccessControl(Object(CFilterChain))
#9 C:\xampplite\htdocs\mypropertylk\protected\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CController.php(292): CFilterChain->run()
#11 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CController.php(266): CController->runActionWithFilters(Object(EditPropertyAction), Array)
#12 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(276): CController->run('editproperty')
#13 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('property/editpr...')
#14 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#15 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#16 {main}
REQUEST_URI=/mypropertylk/property/editproperty?pid=47
HTTP_REFERER=http://localhost/mypropertylk/property/editproperty?pid=47
---
2014/07/24 10:28:34 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[HY000]: General error: 1452 Cannot add or update a child row: a foreign key constraint fails (`mypropertylk`.`property`, CONSTRAINT `property_ibfk_3` FOREIGN KEY (`province`) REFERENCES `province` (`id`) ON DELETE CASCADE). The SQL statement executed was: UPDATE `property` SET `pid`=:yp0, `propcondition`=:yp1, `owner`=:yp2, `agent`=:yp3, `otheragent`=:yp4, `weeklyrent`=:yp5, `monthlyrent`=:yp6, `securebond`=:yp7, `price`=:yp8, `dispalyprice`=:yp9, `availabledate`=:yp10, `vendorname`=:yp11, `vendoremail`=:yp12, `vendorphone`=:yp13, `sendemail`=:yp14, `unitnum`=:yp15, `lotnum`=:yp16, `number`=:yp17, `streetaddress`=:yp18, `areaname`=:yp19, `townname`=:yp20, `hidestreetaddress`=:yp21, `district`=:yp22, `province`=:yp23, `bedrooms`=:yp24, `bathrooms`=:yp25, `ensuites`=:yp26, `toilets`=:yp27, `parkgaragespaces`=:yp28, `parkcarpetspaces`=:yp29, `parkopenspaces`=:yp30, `livingarea`=:yp31, `housesize`=:yp32, `housesquares`=:yp33, `landsize`=:yp34, `landsquares`=:yp35, `floorarea`=:yp36, `floorsquares`=:yp37, `tenuretype`=:yp38, `building`=:yp39, `parkingspaces`=:yp40, `parkcomment`=:yp41, `zoning`=:yp42, `outgoings`=:yp43, `eer`=:yp44, `balcony`=:yp45, `deck`=:yp46, `oea`=:yp47, `shed`=:yp48, `remotegarage`=:yp49, `swimpool`=:yp50, `courtyard`=:yp51, `fullyfenced`=:yp52, `outsidespa`=:yp53, `securepark`=:yp54, `tenniscourt`=:yp55, `spabovroundeg`=:yp56, `alarmsys`=:yp57, `biltinwardrobes`=:yp58, `dvs`=:yp59, `gym`=:yp60, `intercom`=:yp61, `rumpusroom`=:yp62, `workshop`=:yp63, `broadbandinternet`=:yp64, `dishwasher`=:yp65, `floorboards`=:yp66, `insidespa`=:yp67, `paytv`=:yp68, `study`=:yp69, `ac`=:yp70, `heating`=:yp71, `cooling`=:yp72, `solarpower`=:yp73, `solarhotwater`=:yp74, `watertank`=:yp75, `otherfeatures`=:yp76, `headline`=:yp77, `desc`=:yp78, `vediourl`=:yp79, `onlinetour1`=:yp80, `onlinetour2`=:yp81, `entrydate`=:yp82, `status`=:yp83, `type`=:yp84, `pricetype`=:yp85 WHERE `property`.`pid`=47.
in C:\xampplite\htdocs\mypropertylk\protected\controllers\Property\EditPropertyAction.php (64)
in C:\xampplite\htdocs\mypropertylk\protected\controllers\PropertyController.php (74)
in C:\xampplite\htdocs\mypropertylk\index.php (15)
2014/07/24 10:28:34 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[HY000]: General error: 1452 Cannot add or update a child row: a foreign key constraint fails (`mypropertylk`.`property`, CONSTRAINT `property_ibfk_3` FOREIGN KEY (`province`) REFERENCES `province` (`id`) ON DELETE CASCADE). The SQL statement executed was: UPDATE `property` SET `pid`=:yp0, `propcondition`=:yp1, `owner`=:yp2, `agent`=:yp3, `otheragent`=:yp4, `weeklyrent`=:yp5, `monthlyrent`=:yp6, `securebond`=:yp7, `price`=:yp8, `dispalyprice`=:yp9, `availabledate`=:yp10, `vendorname`=:yp11, `vendoremail`=:yp12, `vendorphone`=:yp13, `sendemail`=:yp14, `unitnum`=:yp15, `lotnum`=:yp16, `number`=:yp17, `streetaddress`=:yp18, `areaname`=:yp19, `townname`=:yp20, `hidestreetaddress`=:yp21, `district`=:yp22, `province`=:yp23, `bedrooms`=:yp24, `bathrooms`=:yp25, `ensuites`=:yp26, `toilets`=:yp27, `parkgaragespaces`=:yp28, `parkcarpetspaces`=:yp29, `parkopenspaces`=:yp30, `livingarea`=:yp31, `housesize`=:yp32, `housesquares`=:yp33, `landsize`=:yp34, `landsquares`=:yp35, `floorarea`=:yp36, `floorsquares`=:yp37, `tenuretype`=:yp38, `building`=:yp39, `parkingspaces`=:yp40, `parkcomment`=:yp41, `zoning`=:yp42, `outgoings`=:yp43, `eer`=:yp44, `balcony`=:yp45, `deck`=:yp46, `oea`=:yp47, `shed`=:yp48, `remotegarage`=:yp49, `swimpool`=:yp50, `courtyard`=:yp51, `fullyfenced`=:yp52, `outsidespa`=:yp53, `securepark`=:yp54, `tenniscourt`=:yp55, `spabovroundeg`=:yp56, `alarmsys`=:yp57, `biltinwardrobes`=:yp58, `dvs`=:yp59, `gym`=:yp60, `intercom`=:yp61, `rumpusroom`=:yp62, `workshop`=:yp63, `broadbandinternet`=:yp64, `dishwasher`=:yp65, `floorboards`=:yp66, `insidespa`=:yp67, `paytv`=:yp68, `study`=:yp69, `ac`=:yp70, `heating`=:yp71, `cooling`=:yp72, `solarpower`=:yp73, `solarhotwater`=:yp74, `watertank`=:yp75, `otherfeatures`=:yp76, `headline`=:yp77, `desc`=:yp78, `vediourl`=:yp79, `onlinetour1`=:yp80, `onlinetour2`=:yp81, `entrydate`=:yp82, `status`=:yp83, `type`=:yp84, `pricetype`=:yp85 WHERE `property`.`pid`=47' in C:\xampplite\htdocs\mypropertylk\protected\framework\db\CDbCommand.php:354
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\db\ar\CActiveRecord.php(1681): CDbCommand->execute()
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\db\ar\CActiveRecord.php(1061): CActiveRecord->updateByPk('47', Array)
#2 C:\xampplite\htdocs\mypropertylk\protected\framework\db\ar\CActiveRecord.php(787): CActiveRecord->update(NULL)
#3 C:\xampplite\htdocs\mypropertylk\protected\controllers\Property\EditPropertyAction.php(64): CActiveRecord->save(false)
#4 C:\xampplite\htdocs\mypropertylk\protected\framework\web\actions\CAction.php(75): EditPropertyAction->run()
#5 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CController.php(309): CAction->runWithParams(Array)
#6 C:\xampplite\htdocs\mypropertylk\protected\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(EditPropertyAction))
#7 C:\xampplite\htdocs\mypropertylk\protected\controllers\PropertyController.php(74): CFilterChain->run()
#8 C:\xampplite\htdocs\mypropertylk\protected\framework\web\filters\CInlineFilter.php(59): PropertyController->filterAccessControl(Object(CFilterChain))
#9 C:\xampplite\htdocs\mypropertylk\protected\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CController.php(292): CFilterChain->run()
#11 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CController.php(266): CController->runActionWithFilters(Object(EditPropertyAction), Array)
#12 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(276): CController->run('editproperty')
#13 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('property/editpr...')
#14 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#15 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#16 {main}
REQUEST_URI=/mypropertylk/property/editproperty?pid=47
HTTP_REFERER=http://localhost/mypropertylk/property/editproperty?pid=47
---
2014/07/24 10:36:43 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[HY000]: General error: 1452 Cannot add or update a child row: a foreign key constraint fails (`mypropertylk`.`property`, CONSTRAINT `property_ibfk_2` FOREIGN KEY (`agent`) REFERENCES `user` (`id`) ON DELETE CASCADE). The SQL statement executed was: UPDATE `property` SET `pid`=:yp0, `propcondition`=:yp1, `owner`=:yp2, `agent`=:yp3, `otheragent`=:yp4, `weeklyrent`=:yp5, `monthlyrent`=:yp6, `securebond`=:yp7, `price`=:yp8, `dispalyprice`=:yp9, `availabledate`=:yp10, `vendorname`=:yp11, `vendoremail`=:yp12, `vendorphone`=:yp13, `sendemail`=:yp14, `unitnum`=:yp15, `lotnum`=:yp16, `number`=:yp17, `streetaddress`=:yp18, `areaname`=:yp19, `townname`=:yp20, `hidestreetaddress`=:yp21, `district`=:yp22, `province`=:yp23, `bedrooms`=:yp24, `bathrooms`=:yp25, `ensuites`=:yp26, `toilets`=:yp27, `parkgaragespaces`=:yp28, `parkcarpetspaces`=:yp29, `parkopenspaces`=:yp30, `livingarea`=:yp31, `housesize`=:yp32, `housesquares`=:yp33, `landsize`=:yp34, `landsquares`=:yp35, `floorarea`=:yp36, `floorsquares`=:yp37, `tenuretype`=:yp38, `building`=:yp39, `parkingspaces`=:yp40, `parkcomment`=:yp41, `zoning`=:yp42, `outgoings`=:yp43, `eer`=:yp44, `balcony`=:yp45, `deck`=:yp46, `oea`=:yp47, `shed`=:yp48, `remotegarage`=:yp49, `swimpool`=:yp50, `courtyard`=:yp51, `fullyfenced`=:yp52, `outsidespa`=:yp53, `securepark`=:yp54, `tenniscourt`=:yp55, `spabovroundeg`=:yp56, `alarmsys`=:yp57, `biltinwardrobes`=:yp58, `dvs`=:yp59, `gym`=:yp60, `intercom`=:yp61, `rumpusroom`=:yp62, `workshop`=:yp63, `broadbandinternet`=:yp64, `dishwasher`=:yp65, `floorboards`=:yp66, `insidespa`=:yp67, `paytv`=:yp68, `study`=:yp69, `ac`=:yp70, `heating`=:yp71, `cooling`=:yp72, `solarpower`=:yp73, `solarhotwater`=:yp74, `watertank`=:yp75, `otherfeatures`=:yp76, `headline`=:yp77, `desc`=:yp78, `vediourl`=:yp79, `onlinetour1`=:yp80, `onlinetour2`=:yp81, `entrydate`=:yp82, `status`=:yp83, `type`=:yp84, `pricetype`=:yp85 WHERE `property`.`pid`=47.
in C:\xampplite\htdocs\mypropertylk\protected\controllers\Property\EditPropertyAction.php (64)
in C:\xampplite\htdocs\mypropertylk\protected\controllers\PropertyController.php (74)
in C:\xampplite\htdocs\mypropertylk\index.php (15)
2014/07/24 10:36:43 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[HY000]: General error: 1452 Cannot add or update a child row: a foreign key constraint fails (`mypropertylk`.`property`, CONSTRAINT `property_ibfk_2` FOREIGN KEY (`agent`) REFERENCES `user` (`id`) ON DELETE CASCADE). The SQL statement executed was: UPDATE `property` SET `pid`=:yp0, `propcondition`=:yp1, `owner`=:yp2, `agent`=:yp3, `otheragent`=:yp4, `weeklyrent`=:yp5, `monthlyrent`=:yp6, `securebond`=:yp7, `price`=:yp8, `dispalyprice`=:yp9, `availabledate`=:yp10, `vendorname`=:yp11, `vendoremail`=:yp12, `vendorphone`=:yp13, `sendemail`=:yp14, `unitnum`=:yp15, `lotnum`=:yp16, `number`=:yp17, `streetaddress`=:yp18, `areaname`=:yp19, `townname`=:yp20, `hidestreetaddress`=:yp21, `district`=:yp22, `province`=:yp23, `bedrooms`=:yp24, `bathrooms`=:yp25, `ensuites`=:yp26, `toilets`=:yp27, `parkgaragespaces`=:yp28, `parkcarpetspaces`=:yp29, `parkopenspaces`=:yp30, `livingarea`=:yp31, `housesize`=:yp32, `housesquares`=:yp33, `landsize`=:yp34, `landsquares`=:yp35, `floorarea`=:yp36, `floorsquares`=:yp37, `tenuretype`=:yp38, `building`=:yp39, `parkingspaces`=:yp40, `parkcomment`=:yp41, `zoning`=:yp42, `outgoings`=:yp43, `eer`=:yp44, `balcony`=:yp45, `deck`=:yp46, `oea`=:yp47, `shed`=:yp48, `remotegarage`=:yp49, `swimpool`=:yp50, `courtyard`=:yp51, `fullyfenced`=:yp52, `outsidespa`=:yp53, `securepark`=:yp54, `tenniscourt`=:yp55, `spabovroundeg`=:yp56, `alarmsys`=:yp57, `biltinwardrobes`=:yp58, `dvs`=:yp59, `gym`=:yp60, `intercom`=:yp61, `rumpusroom`=:yp62, `workshop`=:yp63, `broadbandinternet`=:yp64, `dishwasher`=:yp65, `floorboards`=:yp66, `insidespa`=:yp67, `paytv`=:yp68, `study`=:yp69, `ac`=:yp70, `heating`=:yp71, `cooling`=:yp72, `solarpower`=:yp73, `solarhotwater`=:yp74, `watertank`=:yp75, `otherfeatures`=:yp76, `headline`=:yp77, `desc`=:yp78, `vediourl`=:yp79, `onlinetour1`=:yp80, `onlinetour2`=:yp81, `entrydate`=:yp82, `status`=:yp83, `type`=:yp84, `pricetype`=:yp85 WHERE `property`.`pid`=47' in C:\xampplite\htdocs\mypropertylk\protected\framework\db\CDbCommand.php:354
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\db\ar\CActiveRecord.php(1681): CDbCommand->execute()
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\db\ar\CActiveRecord.php(1061): CActiveRecord->updateByPk('47', Array)
#2 C:\xampplite\htdocs\mypropertylk\protected\framework\db\ar\CActiveRecord.php(787): CActiveRecord->update(NULL)
#3 C:\xampplite\htdocs\mypropertylk\protected\controllers\Property\EditPropertyAction.php(64): CActiveRecord->save(false)
#4 C:\xampplite\htdocs\mypropertylk\protected\framework\web\actions\CAction.php(75): EditPropertyAction->run()
#5 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CController.php(309): CAction->runWithParams(Array)
#6 C:\xampplite\htdocs\mypropertylk\protected\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(EditPropertyAction))
#7 C:\xampplite\htdocs\mypropertylk\protected\controllers\PropertyController.php(74): CFilterChain->run()
#8 C:\xampplite\htdocs\mypropertylk\protected\framework\web\filters\CInlineFilter.php(59): PropertyController->filterAccessControl(Object(CFilterChain))
#9 C:\xampplite\htdocs\mypropertylk\protected\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CController.php(292): CFilterChain->run()
#11 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CController.php(266): CController->runActionWithFilters(Object(EditPropertyAction), Array)
#12 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(276): CController->run('editproperty')
#13 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('property/editpr...')
#14 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#15 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#16 {main}
REQUEST_URI=/mypropertylk/property/editproperty?pid=47
HTTP_REFERER=http://localhost/mypropertylk/property/editproperty?pid=47
---
2014/07/24 10:46:01 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[HY000]: General error: 1452 Cannot add or update a child row: a foreign key constraint fails (`mypropertylk`.`property`, CONSTRAINT `property_ibfk_4` FOREIGN KEY (`district`) REFERENCES `district` (`id`) ON DELETE CASCADE). The SQL statement executed was: UPDATE `property` SET `pid`=:yp0, `propcondition`=:yp1, `owner`=:yp2, `agent`=:yp3, `otheragent`=:yp4, `weeklyrent`=:yp5, `monthlyrent`=:yp6, `securebond`=:yp7, `price`=:yp8, `dispalyprice`=:yp9, `availabledate`=:yp10, `vendorname`=:yp11, `vendoremail`=:yp12, `vendorphone`=:yp13, `sendemail`=:yp14, `unitnum`=:yp15, `lotnum`=:yp16, `number`=:yp17, `streetaddress`=:yp18, `areaname`=:yp19, `townname`=:yp20, `hidestreetaddress`=:yp21, `district`=:yp22, `province`=:yp23, `bedrooms`=:yp24, `bathrooms`=:yp25, `ensuites`=:yp26, `toilets`=:yp27, `parkgaragespaces`=:yp28, `parkcarpetspaces`=:yp29, `parkopenspaces`=:yp30, `livingarea`=:yp31, `housesize`=:yp32, `housesquares`=:yp33, `landsize`=:yp34, `landsquares`=:yp35, `floorarea`=:yp36, `floorsquares`=:yp37, `tenuretype`=:yp38, `building`=:yp39, `parkingspaces`=:yp40, `parkcomment`=:yp41, `zoning`=:yp42, `outgoings`=:yp43, `eer`=:yp44, `balcony`=:yp45, `deck`=:yp46, `oea`=:yp47, `shed`=:yp48, `remotegarage`=:yp49, `swimpool`=:yp50, `courtyard`=:yp51, `fullyfenced`=:yp52, `outsidespa`=:yp53, `securepark`=:yp54, `tenniscourt`=:yp55, `spabovroundeg`=:yp56, `alarmsys`=:yp57, `biltinwardrobes`=:yp58, `dvs`=:yp59, `gym`=:yp60, `intercom`=:yp61, `rumpusroom`=:yp62, `workshop`=:yp63, `broadbandinternet`=:yp64, `dishwasher`=:yp65, `floorboards`=:yp66, `insidespa`=:yp67, `paytv`=:yp68, `study`=:yp69, `ac`=:yp70, `heating`=:yp71, `cooling`=:yp72, `solarpower`=:yp73, `solarhotwater`=:yp74, `watertank`=:yp75, `otherfeatures`=:yp76, `headline`=:yp77, `desc`=:yp78, `vediourl`=:yp79, `onlinetour1`=:yp80, `onlinetour2`=:yp81, `entrydate`=:yp82, `status`=:yp83, `type`=:yp84, `pricetype`=:yp85 WHERE `property`.`pid`=46.
in C:\xampplite\htdocs\mypropertylk\protected\controllers\Property\EditPropertyAction.php (64)
in C:\xampplite\htdocs\mypropertylk\protected\controllers\PropertyController.php (74)
in C:\xampplite\htdocs\mypropertylk\index.php (15)
2014/07/24 10:46:01 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[HY000]: General error: 1452 Cannot add or update a child row: a foreign key constraint fails (`mypropertylk`.`property`, CONSTRAINT `property_ibfk_4` FOREIGN KEY (`district`) REFERENCES `district` (`id`) ON DELETE CASCADE). The SQL statement executed was: UPDATE `property` SET `pid`=:yp0, `propcondition`=:yp1, `owner`=:yp2, `agent`=:yp3, `otheragent`=:yp4, `weeklyrent`=:yp5, `monthlyrent`=:yp6, `securebond`=:yp7, `price`=:yp8, `dispalyprice`=:yp9, `availabledate`=:yp10, `vendorname`=:yp11, `vendoremail`=:yp12, `vendorphone`=:yp13, `sendemail`=:yp14, `unitnum`=:yp15, `lotnum`=:yp16, `number`=:yp17, `streetaddress`=:yp18, `areaname`=:yp19, `townname`=:yp20, `hidestreetaddress`=:yp21, `district`=:yp22, `province`=:yp23, `bedrooms`=:yp24, `bathrooms`=:yp25, `ensuites`=:yp26, `toilets`=:yp27, `parkgaragespaces`=:yp28, `parkcarpetspaces`=:yp29, `parkopenspaces`=:yp30, `livingarea`=:yp31, `housesize`=:yp32, `housesquares`=:yp33, `landsize`=:yp34, `landsquares`=:yp35, `floorarea`=:yp36, `floorsquares`=:yp37, `tenuretype`=:yp38, `building`=:yp39, `parkingspaces`=:yp40, `parkcomment`=:yp41, `zoning`=:yp42, `outgoings`=:yp43, `eer`=:yp44, `balcony`=:yp45, `deck`=:yp46, `oea`=:yp47, `shed`=:yp48, `remotegarage`=:yp49, `swimpool`=:yp50, `courtyard`=:yp51, `fullyfenced`=:yp52, `outsidespa`=:yp53, `securepark`=:yp54, `tenniscourt`=:yp55, `spabovroundeg`=:yp56, `alarmsys`=:yp57, `biltinwardrobes`=:yp58, `dvs`=:yp59, `gym`=:yp60, `intercom`=:yp61, `rumpusroom`=:yp62, `workshop`=:yp63, `broadbandinternet`=:yp64, `dishwasher`=:yp65, `floorboards`=:yp66, `insidespa`=:yp67, `paytv`=:yp68, `study`=:yp69, `ac`=:yp70, `heating`=:yp71, `cooling`=:yp72, `solarpower`=:yp73, `solarhotwater`=:yp74, `watertank`=:yp75, `otherfeatures`=:yp76, `headline`=:yp77, `desc`=:yp78, `vediourl`=:yp79, `onlinetour1`=:yp80, `onlinetour2`=:yp81, `entrydate`=:yp82, `status`=:yp83, `type`=:yp84, `pricetype`=:yp85 WHERE `property`.`pid`=46' in C:\xampplite\htdocs\mypropertylk\protected\framework\db\CDbCommand.php:354
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\db\ar\CActiveRecord.php(1681): CDbCommand->execute()
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\db\ar\CActiveRecord.php(1061): CActiveRecord->updateByPk('46', Array)
#2 C:\xampplite\htdocs\mypropertylk\protected\framework\db\ar\CActiveRecord.php(787): CActiveRecord->update(NULL)
#3 C:\xampplite\htdocs\mypropertylk\protected\controllers\Property\EditPropertyAction.php(64): CActiveRecord->save(false)
#4 C:\xampplite\htdocs\mypropertylk\protected\framework\web\actions\CAction.php(75): EditPropertyAction->run()
#5 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CController.php(309): CAction->runWithParams(Array)
#6 C:\xampplite\htdocs\mypropertylk\protected\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(EditPropertyAction))
#7 C:\xampplite\htdocs\mypropertylk\protected\controllers\PropertyController.php(74): CFilterChain->run()
#8 C:\xampplite\htdocs\mypropertylk\protected\framework\web\filters\CInlineFilter.php(59): PropertyController->filterAccessControl(Object(CFilterChain))
#9 C:\xampplite\htdocs\mypropertylk\protected\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CController.php(292): CFilterChain->run()
#11 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CController.php(266): CController->runActionWithFilters(Object(EditPropertyAction), Array)
#12 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(276): CController->run('editproperty')
#13 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('property/editpr...')
#14 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#15 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#16 {main}
REQUEST_URI=/mypropertylk/property/editproperty?pid=46
HTTP_REFERER=http://localhost/mypropertylk/property/editproperty?pid=46
---
2014/07/24 10:59:32 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "mypropertylk/property/editproperty".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('mypropertylk/pr...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/mypropertylk/property/editproperty?pid=47
HTTP_REFERER=http://localhost/mypropertylk/login
---
2014/07/24 10:59:42 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "mypropertylk/property/editproperty".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('mypropertylk/pr...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/mypropertylk/property/editproperty?pid=47
HTTP_REFERER=http://localhost/mypropertylk/login
---
2014/07/24 10:59:52 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "mypropertylk/property/editproperty".' in C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampplite\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('mypropertylk/pr...')
#1 C:\xampplite\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/mypropertylk/property/editproperty?pid=47
HTTP_REFERER=http://localhost/mypropertylk/login
---
2014/07/24 05:30:01 [warning] [application] Failed to set unsafe attribute "adsample" of "Adprice".
in C:\xampp\htdocs\mypropertylk\protected\controllers\Advertising\AddpricesAction.php (24)
in C:\xampp\htdocs\mypropertylk\protected\controllers\AdvertisingController.php (69)
in C:\xampp\htdocs\mypropertylk\index.php (15)
2014/07/24 06:03:03 [error] [exception.CException] exception 'CException' with message 'Property "Adprice.size0->size" is not defined.' in C:\xampp\htdocs\mypropertylk\protected\framework\base\CComponent.php:131
Stack trace:
#0 C:\xampp\htdocs\mypropertylk\protected\framework\db\ar\CActiveRecord.php(144): CComponent->__get('size0->size')
#1 C:\xampp\htdocs\mypropertylk\protected\framework\web\helpers\CHtml.php(1696): CActiveRecord->__get('size0->size')
#2 C:\xampp\htdocs\mypropertylk\protected\framework\web\helpers\CHtml.php(1660): CHtml::value(Object(Adprice), 'size0->size')
#3 C:\xampp\htdocs\mypropertylk\protected\views\advertising\editadvertisement.php(143): CHtml::listData(Array, 'size', 'size0->size')
#4 C:\xampp\htdocs\mypropertylk\protected\framework\web\CBaseController.php(127): require('C:\\xampp\\htdocs...')
#5 C:\xampp\htdocs\mypropertylk\protected\framework\web\CBaseController.php(96): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#6 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(870): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#7 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(783): CController->renderPartial('editadvertiseme...', Array, true)
#8 C:\xampp\htdocs\mypropertylk\protected\controllers\Advertising\EditAdvertisementAction.php(71): CController->render('editadvertiseme...', Array)
#9 C:\xampp\htdocs\mypropertylk\protected\framework\web\actions\CAction.php(75): EditAdvertisementAction->run()
#10 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(309): CAction->runWithParams(Array)
#11 C:\xampp\htdocs\mypropertylk\protected\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(EditAdvertisementAction))
#12 C:\xampp\htdocs\mypropertylk\protected\controllers\AdvertisingController.php(69): CFilterChain->run()
#13 C:\xampp\htdocs\mypropertylk\protected\framework\web\filters\CInlineFilter.php(59): AdvertisingController->filterAccessControl(Object(CFilterChain))
#14 C:\xampp\htdocs\mypropertylk\protected\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#15 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(292): CFilterChain->run()
#16 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(266): CController->runActionWithFilters(Object(EditAdvertisementAction), Array)
#17 C:\xampp\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(276): CController->run('editadvertiseme...')
#18 C:\xampp\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('advertising/edi...')
#19 C:\xampp\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#20 C:\xampp\htdocs\mypropertylk\index.php(15): CApplication->run()
#21 {main}
REQUEST_URI=/mypropertylk/advertising/editadvertisement/id/44
HTTP_REFERER=http://localhost/mypropertylk/advertising/advertisement
---
2014/07/24 06:13:07 [error] [exception.CException] exception 'CException' with message 'Property "Adprice.size0->size" is not defined.' in C:\xampp\htdocs\mypropertylk\protected\framework\base\CComponent.php:131
Stack trace:
#0 C:\xampp\htdocs\mypropertylk\protected\framework\db\ar\CActiveRecord.php(144): CComponent->__get('size0->size')
#1 C:\xampp\htdocs\mypropertylk\protected\framework\web\helpers\CHtml.php(1696): CActiveRecord->__get('size0->size')
#2 C:\xampp\htdocs\mypropertylk\protected\framework\web\helpers\CHtml.php(1660): CHtml::value(Object(Adprice), 'size0->size')
#3 C:\xampp\htdocs\mypropertylk\protected\views\advertising\editadvertisement.php(143): CHtml::listData(Array, 'size', 'size0->size')
#4 C:\xampp\htdocs\mypropertylk\protected\framework\web\CBaseController.php(127): require('C:\\xampp\\htdocs...')
#5 C:\xampp\htdocs\mypropertylk\protected\framework\web\CBaseController.php(96): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#6 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(870): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#7 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(783): CController->renderPartial('editadvertiseme...', Array, true)
#8 C:\xampp\htdocs\mypropertylk\protected\controllers\Advertising\EditAdvertisementAction.php(71): CController->render('editadvertiseme...', Array)
#9 C:\xampp\htdocs\mypropertylk\protected\framework\web\actions\CAction.php(75): EditAdvertisementAction->run()
#10 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(309): CAction->runWithParams(Array)
#11 C:\xampp\htdocs\mypropertylk\protected\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(EditAdvertisementAction))
#12 C:\xampp\htdocs\mypropertylk\protected\controllers\AdvertisingController.php(69): CFilterChain->run()
#13 C:\xampp\htdocs\mypropertylk\protected\framework\web\filters\CInlineFilter.php(59): AdvertisingController->filterAccessControl(Object(CFilterChain))
#14 C:\xampp\htdocs\mypropertylk\protected\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#15 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(292): CFilterChain->run()
#16 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(266): CController->runActionWithFilters(Object(EditAdvertisementAction), Array)
#17 C:\xampp\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(276): CController->run('editadvertiseme...')
#18 C:\xampp\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('advertising/edi...')
#19 C:\xampp\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#20 C:\xampp\htdocs\mypropertylk\index.php(15): CApplication->run()
#21 {main}
REQUEST_URI=/mypropertylk/advertising/editadvertisement/id/44
HTTP_REFERER=http://localhost/mypropertylk/advertising/advertisement
---
2014/07/24 06:28:34 [error] [php] Object of class Adprice could not be converted to string (C:\xampp\htdocs\mypropertylk\protected\views\advertising\editadvertisement.php:180)
Stack trace:
#0 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(870): AdvertisingController->renderFile()
#1 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(783): AdvertisingController->renderPartial()
#2 C:\xampp\htdocs\mypropertylk\protected\controllers\Advertising\EditAdvertisementAction.php(71): AdvertisingController->render()
#3 C:\xampp\htdocs\mypropertylk\protected\framework\web\actions\CAction.php(75): EditAdvertisementAction->run()
#4 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(309): EditAdvertisementAction->runWithParams()
#5 C:\xampp\htdocs\mypropertylk\protected\framework\web\filters\CFilterChain.php(134): AdvertisingController->runAction()
#6 C:\xampp\htdocs\mypropertylk\protected\controllers\AdvertisingController.php(69): CFilterChain->run()
#7 C:\xampp\htdocs\mypropertylk\protected\framework\web\filters\CInlineFilter.php(59): AdvertisingController->filterAccessControl()
#8 C:\xampp\htdocs\mypropertylk\protected\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter()
#9 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(292): CFilterChain->run()
#10 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(266): AdvertisingController->runActionWithFilters()
#11 C:\xampp\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(276): AdvertisingController->run()
#12 C:\xampp\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController()
#13 C:\xampp\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#14 C:\xampp\htdocs\mypropertylk\index.php(15): CWebApplication->run()
REQUEST_URI=/mypropertylk/advertising/editadvertisement/id/44
in C:\xampp\htdocs\mypropertylk\protected\views\advertising\editadvertisement.php (180)
in C:\xampp\htdocs\mypropertylk\protected\controllers\Advertising\EditAdvertisementAction.php (71)
in C:\xampp\htdocs\mypropertylk\protected\controllers\AdvertisingController.php (69)
2014/07/24 06:30:26 [error] [php] Object of class Adprice could not be converted to string (C:\xampp\htdocs\mypropertylk\protected\views\advertising\editadvertisement.php:180)
Stack trace:
#0 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(870): AdvertisingController->renderFile()
#1 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(783): AdvertisingController->renderPartial()
#2 C:\xampp\htdocs\mypropertylk\protected\controllers\Advertising\EditAdvertisementAction.php(71): AdvertisingController->render()
#3 C:\xampp\htdocs\mypropertylk\protected\framework\web\actions\CAction.php(75): EditAdvertisementAction->run()
#4 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(309): EditAdvertisementAction->runWithParams()
#5 C:\xampp\htdocs\mypropertylk\protected\framework\web\filters\CFilterChain.php(134): AdvertisingController->runAction()
#6 C:\xampp\htdocs\mypropertylk\protected\controllers\AdvertisingController.php(69): CFilterChain->run()
#7 C:\xampp\htdocs\mypropertylk\protected\framework\web\filters\CInlineFilter.php(59): AdvertisingController->filterAccessControl()
#8 C:\xampp\htdocs\mypropertylk\protected\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter()
#9 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(292): CFilterChain->run()
#10 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(266): AdvertisingController->runActionWithFilters()
#11 C:\xampp\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(276): AdvertisingController->run()
#12 C:\xampp\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController()
#13 C:\xampp\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#14 C:\xampp\htdocs\mypropertylk\index.php(15): CWebApplication->run()
REQUEST_URI=/mypropertylk/advertising/editadvertisement/id/44
in C:\xampp\htdocs\mypropertylk\protected\views\advertising\editadvertisement.php (180)
in C:\xampp\htdocs\mypropertylk\protected\controllers\Advertising\EditAdvertisementAction.php (71)
in C:\xampp\htdocs\mypropertylk\protected\controllers\AdvertisingController.php (69)
2014/07/24 06:49:03 [error] [php] Undefined variable: price (C:\xampp\htdocs\mypropertylk\protected\views\advertising\editadvertisement.php:178)
Stack trace:
#0 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(870): AdvertisingController->renderFile()
#1 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(783): AdvertisingController->renderPartial()
#2 C:\xampp\htdocs\mypropertylk\protected\controllers\Advertising\EditAdvertisementAction.php(71): AdvertisingController->render()
#3 C:\xampp\htdocs\mypropertylk\protected\framework\web\actions\CAction.php(75): EditAdvertisementAction->run()
#4 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(309): EditAdvertisementAction->runWithParams()
#5 C:\xampp\htdocs\mypropertylk\protected\framework\web\filters\CFilterChain.php(134): AdvertisingController->runAction()
#6 C:\xampp\htdocs\mypropertylk\protected\controllers\AdvertisingController.php(69): CFilterChain->run()
#7 C:\xampp\htdocs\mypropertylk\protected\framework\web\filters\CInlineFilter.php(59): AdvertisingController->filterAccessControl()
#8 C:\xampp\htdocs\mypropertylk\protected\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter()
#9 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(292): CFilterChain->run()
#10 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(266): AdvertisingController->runActionWithFilters()
#11 C:\xampp\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(276): AdvertisingController->run()
#12 C:\xampp\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController()
#13 C:\xampp\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#14 C:\xampp\htdocs\mypropertylk\index.php(15): CWebApplication->run()
REQUEST_URI=/mypropertylk/advertising/editadvertisement/id/44
in C:\xampp\htdocs\mypropertylk\protected\views\advertising\editadvertisement.php (178)
in C:\xampp\htdocs\mypropertylk\protected\controllers\Advertising\EditAdvertisementAction.php (71)
in C:\xampp\htdocs\mypropertylk\protected\controllers\AdvertisingController.php (69)
2014/07/24 06:50:15 [error] [php] Undefined variable: adprice (C:\xampp\htdocs\mypropertylk\protected\views\advertising\editadvertisement.php:178)
Stack trace:
#0 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(870): AdvertisingController->renderFile()
#1 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(783): AdvertisingController->renderPartial()
#2 C:\xampp\htdocs\mypropertylk\protected\controllers\Advertising\EditAdvertisementAction.php(71): AdvertisingController->render()
#3 C:\xampp\htdocs\mypropertylk\protected\framework\web\actions\CAction.php(75): EditAdvertisementAction->run()
#4 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(309): EditAdvertisementAction->runWithParams()
#5 C:\xampp\htdocs\mypropertylk\protected\framework\web\filters\CFilterChain.php(134): AdvertisingController->runAction()
#6 C:\xampp\htdocs\mypropertylk\protected\controllers\AdvertisingController.php(69): CFilterChain->run()
#7 C:\xampp\htdocs\mypropertylk\protected\framework\web\filters\CInlineFilter.php(59): AdvertisingController->filterAccessControl()
#8 C:\xampp\htdocs\mypropertylk\protected\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter()
#9 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(292): CFilterChain->run()
#10 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(266): AdvertisingController->runActionWithFilters()
#11 C:\xampp\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(276): AdvertisingController->run()
#12 C:\xampp\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController()
#13 C:\xampp\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#14 C:\xampp\htdocs\mypropertylk\index.php(15): CWebApplication->run()
REQUEST_URI=/mypropertylk/advertising/editadvertisement/id/44
in C:\xampp\htdocs\mypropertylk\protected\views\advertising\editadvertisement.php (178)
in C:\xampp\htdocs\mypropertylk\protected\controllers\Advertising\EditAdvertisementAction.php (71)
in C:\xampp\htdocs\mypropertylk\protected\controllers\AdvertisingController.php (69)
2014/07/24 06:54:37 [error] [system.db.CDbCommand] CDbCommand::fetch() failed: SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'size = 3 LIMIT 1' at line 1. The SQL statement executed was: SELECT * FROM `adprice` `t` WHERE page = 1AND size = 3 LIMIT 1.
in C:\xampp\htdocs\mypropertylk\protected\controllers\Advertising\EditAdvertisementAction.php (23)
in C:\xampp\htdocs\mypropertylk\protected\controllers\AdvertisingController.php (69)
in C:\xampp\htdocs\mypropertylk\index.php (15)
2014/07/24 06:54:37 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'size = 3 LIMIT 1' at line 1. The SQL statement executed was: SELECT * FROM `adprice` `t` WHERE page = 1AND size = 3 LIMIT 1' in C:\xampp\htdocs\mypropertylk\protected\framework\db\CDbCommand.php:528
Stack trace:
#0 C:\xampp\htdocs\mypropertylk\protected\framework\db\CDbCommand.php(408): CDbCommand->queryInternal('fetch', Array, Array)
#1 C:\xampp\htdocs\mypropertylk\protected\framework\db\ar\CActiveRecord.php(1287): CDbCommand->queryRow()
#2 C:\xampp\htdocs\mypropertylk\protected\framework\db\ar\CActiveRecord.php(1392): CActiveRecord->query(Object(CDbCriteria))
#3 C:\xampp\htdocs\mypropertylk\protected\controllers\Advertising\EditAdvertisementAction.php(23): CActiveRecord->find('page = 1AND siz...')
#4 C:\xampp\htdocs\mypropertylk\protected\framework\web\actions\CAction.php(75): EditAdvertisementAction->run()
#5 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(309): CAction->runWithParams(Array)
#6 C:\xampp\htdocs\mypropertylk\protected\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(EditAdvertisementAction))
#7 C:\xampp\htdocs\mypropertylk\protected\controllers\AdvertisingController.php(69): CFilterChain->run()
#8 C:\xampp\htdocs\mypropertylk\protected\framework\web\filters\CInlineFilter.php(59): AdvertisingController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\mypropertylk\protected\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(292): CFilterChain->run()
#11 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(266): CController->runActionWithFilters(Object(EditAdvertisementAction), Array)
#12 C:\xampp\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(276): CController->run('editadvertiseme...')
#13 C:\xampp\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('advertising/edi...')
#14 C:\xampp\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#15 C:\xampp\htdocs\mypropertylk\index.php(15): CApplication->run()
#16 {main}
REQUEST_URI=/mypropertylk/advertising/editadvertisement/id/44
HTTP_REFERER=http://localhost/mypropertylk/advertising/advertisement
---
2014/07/24 07:44:59 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "editadvertisement44".' in C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php:484
Stack trace:
#0 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(271): CController->missingAction('editadvertiseme...')
#1 C:\xampp\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(276): CController->run('editadvertiseme...')
#2 C:\xampp\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('advertising/edi...')
#3 C:\xampp\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#4 C:\xampp\htdocs\mypropertylk\index.php(15): CApplication->run()
#5 {main}
REQUEST_URI=/mypropertylk/advertising/editadvertisement44
HTTP_REFERER=http://localhost/mypropertylk/advertising/editadvertisement/id/44
---
2014/07/24 12:15:57 [error] [exception.CException] exception 'CException' with message 'AdvertisingController cannot find the requested view "editadvertisement".' in C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php:879
Stack trace:
#0 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(783): CController->renderPartial('editadvertiseme...', Array, true)
#1 C:\xampp\htdocs\mypropertylk\protected\controllers\Advertising\EditAdvertisementAction.php(111): CController->render('editadvertiseme...', Array)
#2 C:\xampp\htdocs\mypropertylk\protected\framework\web\actions\CAction.php(75): EditAdvertisementAction->run()
#3 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(309): CAction->runWithParams(Array)
#4 C:\xampp\htdocs\mypropertylk\protected\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(EditAdvertisementAction))
#5 C:\xampp\htdocs\mypropertylk\protected\controllers\AdvertisingController.php(69): CFilterChain->run()
#6 C:\xampp\htdocs\mypropertylk\protected\framework\web\filters\CInlineFilter.php(59): AdvertisingController->filterAccessControl(Object(CFilterChain))
#7 C:\xampp\htdocs\mypropertylk\protected\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(292): CFilterChain->run()
#9 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(266): CController->runActionWithFilters(Object(EditAdvertisementAction), Array)
#10 C:\xampp\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(276): CController->run('editadvertiseme...')
#11 C:\xampp\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('advertising/edi...')
#12 C:\xampp\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#13 C:\xampp\htdocs\mypropertylk\index.php(15): CApplication->run()
#14 {main}
REQUEST_URI=/mypropertylk/advertising/editadvertisement/id/44
HTTP_REFERER=http://localhost/mypropertylk/advertising/advertisement
---
2014/07/24 12:36:39 [error] [system.db.CDbCommand] CDbCommand::fetch() failed: SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'id = 1 LIMIT 1' at line 1. The SQL statement executed was: SELECT * FROM `user` `t` WHERE id = 2OR id = 1 LIMIT 1.
in C:\xampp\htdocs\mypropertylk\protected\controllers\Advertising\EditpricesAction.php (23)
in C:\xampp\htdocs\mypropertylk\protected\controllers\AdvertisingController.php (69)
in C:\xampp\htdocs\mypropertylk\index.php (15)
2014/07/24 12:36:39 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'id = 1 LIMIT 1' at line 1. The SQL statement executed was: SELECT * FROM `user` `t` WHERE id = 2OR id = 1 LIMIT 1' in C:\xampp\htdocs\mypropertylk\protected\framework\db\CDbCommand.php:528
Stack trace:
#0 C:\xampp\htdocs\mypropertylk\protected\framework\db\CDbCommand.php(408): CDbCommand->queryInternal('fetch', Array, Array)
#1 C:\xampp\htdocs\mypropertylk\protected\framework\db\ar\CActiveRecord.php(1287): CDbCommand->queryRow()
#2 C:\xampp\htdocs\mypropertylk\protected\framework\db\ar\CActiveRecord.php(1392): CActiveRecord->query(Object(CDbCriteria))
#3 C:\xampp\htdocs\mypropertylk\protected\controllers\Advertising\EditpricesAction.php(23): CActiveRecord->find('id = 2OR id = 1')
#4 C:\xampp\htdocs\mypropertylk\protected\framework\web\actions\CAction.php(75): EditpricesAction->run()
#5 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(309): CAction->runWithParams(Array)
#6 C:\xampp\htdocs\mypropertylk\protected\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(EditpricesAction))
#7 C:\xampp\htdocs\mypropertylk\protected\controllers\AdvertisingController.php(69): CFilterChain->run()
#8 C:\xampp\htdocs\mypropertylk\protected\framework\web\filters\CInlineFilter.php(59): AdvertisingController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\mypropertylk\protected\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(292): CFilterChain->run()
#11 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(266): CController->runActionWithFilters(Object(EditpricesAction), Array)
#12 C:\xampp\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(276): CController->run('editprices')
#13 C:\xampp\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('advertising/edi...')
#14 C:\xampp\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#15 C:\xampp\htdocs\mypropertylk\index.php(15): CApplication->run()
#16 {main}
REQUEST_URI=/mypropertylk/advertising/editprices/id/2
HTTP_REFERER=http://localhost/mypropertylk/advertising/adpricelisting
---
2014/07/24 12:48:58 [error] [php] Trying to get property of non-object (C:\xampp\htdocs\mypropertylk\protected\views\advertising\editadvertisement.php:180)
Stack trace:
#0 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(870): AdvertisingController->renderFile()
#1 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(783): AdvertisingController->renderPartial()
#2 C:\xampp\htdocs\mypropertylk\protected\controllers\Advertising\EditAdvertisementAction.php(112): AdvertisingController->render()
#3 C:\xampp\htdocs\mypropertylk\protected\framework\web\actions\CAction.php(75): EditAdvertisementAction->run()
#4 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(309): EditAdvertisementAction->runWithParams()
#5 C:\xampp\htdocs\mypropertylk\protected\framework\web\filters\CFilterChain.php(134): AdvertisingController->runAction()
#6 C:\xampp\htdocs\mypropertylk\protected\controllers\AdvertisingController.php(69): CFilterChain->run()
#7 C:\xampp\htdocs\mypropertylk\protected\framework\web\filters\CInlineFilter.php(59): AdvertisingController->filterAccessControl()
#8 C:\xampp\htdocs\mypropertylk\protected\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter()
#9 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(292): CFilterChain->run()
#10 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(266): AdvertisingController->runActionWithFilters()
#11 C:\xampp\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(276): AdvertisingController->run()
#12 C:\xampp\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController()
#13 C:\xampp\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#14 C:\xampp\htdocs\mypropertylk\index.php(15): CWebApplication->run()
REQUEST_URI=/mypropertylk/advertising/editadvertisement/id/41
in C:\xampp\htdocs\mypropertylk\protected\views\advertising\editadvertisement.php (180)
in C:\xampp\htdocs\mypropertylk\protected\controllers\Advertising\EditAdvertisementAction.php (112)
in C:\xampp\htdocs\mypropertylk\protected\controllers\AdvertisingController.php (69)
2014/07/24 14:36:02 [error] [php] Trying to get property of non-object (C:\xampp\htdocs\mypropertylk\protected\views\advertising\editadvertisement.php:180)
Stack trace:
#0 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(870): AdvertisingController->renderFile()
#1 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(783): AdvertisingController->renderPartial()
#2 C:\xampp\htdocs\mypropertylk\protected\controllers\Advertising\EditAdvertisementAction.php(112): AdvertisingController->render()
#3 C:\xampp\htdocs\mypropertylk\protected\framework\web\actions\CAction.php(75): EditAdvertisementAction->run()
#4 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(309): EditAdvertisementAction->runWithParams()
#5 C:\xampp\htdocs\mypropertylk\protected\framework\web\filters\CFilterChain.php(134): AdvertisingController->runAction()
#6 C:\xampp\htdocs\mypropertylk\protected\controllers\AdvertisingController.php(69): CFilterChain->run()
#7 C:\xampp\htdocs\mypropertylk\protected\framework\web\filters\CInlineFilter.php(59): AdvertisingController->filterAccessControl()
#8 C:\xampp\htdocs\mypropertylk\protected\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter()
#9 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(292): CFilterChain->run()
#10 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(266): AdvertisingController->runActionWithFilters()
#11 C:\xampp\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(276): AdvertisingController->run()
#12 C:\xampp\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController()
#13 C:\xampp\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#14 C:\xampp\htdocs\mypropertylk\index.php(15): CWebApplication->run()
REQUEST_URI=/mypropertylk/advertising/editadvertisement/id/48
in C:\xampp\htdocs\mypropertylk\protected\views\advertising\editadvertisement.php (180)
in C:\xampp\htdocs\mypropertylk\protected\controllers\Advertising\EditAdvertisementAction.php (112)
in C:\xampp\htdocs\mypropertylk\protected\controllers\AdvertisingController.php (69)
2014/07/25 05:36:24 [error] [php] Trying to get property of non-object (C:\xampp\htdocs\mypropertylk\protected\views\advertising\editadvertisement.php:180)
Stack trace:
#0 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(870): AdvertisingController->renderFile()
#1 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(783): AdvertisingController->renderPartial()
#2 C:\xampp\htdocs\mypropertylk\protected\controllers\Advertising\EditAdvertisementAction.php(112): AdvertisingController->render()
#3 C:\xampp\htdocs\mypropertylk\protected\framework\web\actions\CAction.php(75): EditAdvertisementAction->run()
#4 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(309): EditAdvertisementAction->runWithParams()
#5 C:\xampp\htdocs\mypropertylk\protected\framework\web\filters\CFilterChain.php(134): AdvertisingController->runAction()
#6 C:\xampp\htdocs\mypropertylk\protected\controllers\AdvertisingController.php(69): CFilterChain->run()
#7 C:\xampp\htdocs\mypropertylk\protected\framework\web\filters\CInlineFilter.php(59): AdvertisingController->filterAccessControl()
#8 C:\xampp\htdocs\mypropertylk\protected\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter()
#9 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(292): CFilterChain->run()
#10 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(266): AdvertisingController->runActionWithFilters()
#11 C:\xampp\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(276): AdvertisingController->run()
#12 C:\xampp\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController()
#13 C:\xampp\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#14 C:\xampp\htdocs\mypropertylk\index.php(15): CWebApplication->run()
REQUEST_URI=/mypropertylk/advertising/editadvertisement/id/17
in C:\xampp\htdocs\mypropertylk\protected\views\advertising\editadvertisement.php (180)
in C:\xampp\htdocs\mypropertylk\protected\controllers\Advertising\EditAdvertisementAction.php (112)
in C:\xampp\htdocs\mypropertylk\protected\controllers\AdvertisingController.php (69)
2014/07/25 06:06:09 [error] [php] Undefined variable: adPrice (C:\xampp\htdocs\mypropertylk\protected\controllers\Advertising\AddAdvertisementAction.php:138)
Stack trace:
#0 C:\xampp\htdocs\mypropertylk\protected\framework\web\filters\CFilterChain.php(134): AdvertisingController->runAction()
#1 C:\xampp\htdocs\mypropertylk\protected\controllers\AdvertisingController.php(69): CFilterChain->run()
#2 C:\xampp\htdocs\mypropertylk\protected\framework\web\filters\CInlineFilter.php(59): AdvertisingController->filterAccessControl()
#3 C:\xampp\htdocs\mypropertylk\protected\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter()
#4 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(292): CFilterChain->run()
#5 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(266): AdvertisingController->runActionWithFilters()
#6 C:\xampp\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(276): AdvertisingController->run()
#7 C:\xampp\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController()
#8 C:\xampp\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#9 C:\xampp\htdocs\mypropertylk\index.php(15): CWebApplication->run()
REQUEST_URI=/mypropertylk/advertising/addadvertisement
in C:\xampp\htdocs\mypropertylk\protected\controllers\Advertising\AddAdvertisementAction.php (138)
in C:\xampp\htdocs\mypropertylk\protected\controllers\AdvertisingController.php (69)
in C:\xampp\htdocs\mypropertylk\index.php (15)
2014/07/25 06:07:09 [error] [php] Trying to get property of non-object (C:\xampp\htdocs\mypropertylk\protected\views\advertising\addadvertisement.php:226)
Stack trace:
#0 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(870): AdvertisingController->renderFile()
#1 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(783): AdvertisingController->renderPartial()
#2 C:\xampp\htdocs\mypropertylk\protected\controllers\Advertising\AddAdvertisementAction.php(139): AdvertisingController->render()
#3 C:\xampp\htdocs\mypropertylk\protected\framework\web\actions\CAction.php(75): AddAdvertisementAction->run()
#4 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(309): AddAdvertisementAction->runWithParams()
#5 C:\xampp\htdocs\mypropertylk\protected\framework\web\filters\CFilterChain.php(134): AdvertisingController->runAction()
#6 C:\xampp\htdocs\mypropertylk\protected\controllers\AdvertisingController.php(69): CFilterChain->run()
#7 C:\xampp\htdocs\mypropertylk\protected\framework\web\filters\CInlineFilter.php(59): AdvertisingController->filterAccessControl()
#8 C:\xampp\htdocs\mypropertylk\protected\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter()
#9 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(292): CFilterChain->run()
#10 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(266): AdvertisingController->runActionWithFilters()
#11 C:\xampp\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(276): AdvertisingController->run()
#12 C:\xampp\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController()
#13 C:\xampp\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#14 C:\xampp\htdocs\mypropertylk\index.php(15): CWebApplication->run()
REQUEST_URI=/mypropertylk/advertising/addadvertisement
in C:\xampp\htdocs\mypropertylk\protected\views\advertising\addadvertisement.php (226)
in C:\xampp\htdocs\mypropertylk\protected\controllers\Advertising\AddAdvertisementAction.php (139)
in C:\xampp\htdocs\mypropertylk\protected\controllers\AdvertisingController.php (69)
2014/07/25 06:16:17 [error] [php] Undefined variable: model (C:\xampp\htdocs\mypropertylk\protected\views\advertising\addadvertisement.php:135)
Stack trace:
#0 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(870): AdvertisingController->renderFile()
#1 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(783): AdvertisingController->renderPartial()
#2 C:\xampp\htdocs\mypropertylk\protected\controllers\Advertising\AddAdvertisementAction.php(58): AdvertisingController->render()
#3 C:\xampp\htdocs\mypropertylk\protected\framework\web\actions\CAction.php(75): AddAdvertisementAction->run()
#4 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(309): AddAdvertisementAction->runWithParams()
#5 C:\xampp\htdocs\mypropertylk\protected\framework\web\filters\CFilterChain.php(134): AdvertisingController->runAction()
#6 C:\xampp\htdocs\mypropertylk\protected\controllers\AdvertisingController.php(69): CFilterChain->run()
#7 C:\xampp\htdocs\mypropertylk\protected\framework\web\filters\CInlineFilter.php(59): AdvertisingController->filterAccessControl()
#8 C:\xampp\htdocs\mypropertylk\protected\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter()
#9 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(292): CFilterChain->run()
#10 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(266): AdvertisingController->runActionWithFilters()
#11 C:\xampp\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(276): AdvertisingController->run()
#12 C:\xampp\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController()
#13 C:\xampp\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#14 C:\xampp\htdocs\mypropertylk\index.php(15): CWebApplication->run()
REQUEST_URI=/mypropertylk/advertising/addadvertisement
in C:\xampp\htdocs\mypropertylk\protected\views\advertising\addadvertisement.php (135)
in C:\xampp\htdocs\mypropertylk\protected\controllers\Advertising\AddAdvertisementAction.php (58)
in C:\xampp\htdocs\mypropertylk\protected\controllers\AdvertisingController.php (69)
2014/07/26 14:31:08 [error] [php] Undefined property: stdClass::$type (C:\xampp\htdocs\mypropertylk\protected\views\property\propertylisting.php:123)
Stack trace:
#0 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(870): PropertyController->renderFile()
#1 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(783): PropertyController->renderPartial()
#2 C:\xampp\htdocs\mypropertylk\protected\controllers\Property\PropertyListingAction.php(68): PropertyController->render()
#3 C:\xampp\htdocs\mypropertylk\protected\framework\web\actions\CAction.php(75): PropertyListingAction->run()
#4 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(309): PropertyListingAction->runWithParams()
#5 C:\xampp\htdocs\mypropertylk\protected\framework\web\filters\CFilterChain.php(134): PropertyController->runAction()
#6 C:\xampp\htdocs\mypropertylk\protected\controllers\PropertyController.php(74): CFilterChain->run()
#7 C:\xampp\htdocs\mypropertylk\protected\framework\web\filters\CInlineFilter.php(59): PropertyController->filterAccessControl()
#8 C:\xampp\htdocs\mypropertylk\protected\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter()
#9 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(292): CFilterChain->run()
#10 C:\xampp\htdocs\mypropertylk\protected\framework\web\CController.php(266): PropertyController->runActionWithFilters()
#11 C:\xampp\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(276): PropertyController->run()
#12 C:\xampp\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController()
#13 C:\xampp\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#14 C:\xampp\htdocs\mypropertylk\index.php(15): CWebApplication->run()
REQUEST_URI=/mypropertylk/property/propertylisting?propertyid=47
in C:\xampp\htdocs\mypropertylk\protected\views\property\propertylisting.php (123)
in C:\xampp\htdocs\mypropertylk\protected\controllers\Property\PropertyListingAction.php (68)
in C:\xampp\htdocs\mypropertylk\protected\controllers\PropertyController.php (74)
